src/types/domain/Commander.ts

```diff
@@ -6,5 +6,6 @@ export type Commander = {
      * A collection of match IDs of matches that the commander is played in
      */
     matches: string[];
```

Comment:
same comment here as in Player.ts to clarify why matches.lenght !== validMatchCount

Reply:
Clarified.